src/Knp/Rad/ResourceResolver/EventListener/ResourcesListener.php

```diff
@@ -105,4 +112,12 @@ protected function castParameter($parameter)
             }
         }
     }
+
+    protected function isNullValue($resource)
+    {
+        return null === $resource ||
```

Comment:
Oh BTW, usually you don't want emtpy arrays te return a 404.
If the thing returns an array, it means you are listing something, right?
You want to display this list anyway (even if empty), not throw a 404.

So I guess we should only check emptiness of non-array things.